stitcher/test/gtest/images.cpp

```diff
@@ -0,0 +1,201 @@
+#include "gtest/gtest.h"
+#include "airmap/images.h"
+#include "airmap/logger.h"
+#include "airmap/panorama.h"
+
+using airmap::stitcher::GeoImage;
+using airmap::stitcher::Panorama;
+using airmap::stitcher::SourceImages;
+using airmap::filesystem::path;
+using airmap::logging::Logger;
+using airmap::logging::stdoe_logger;
+
+class SourceImagesTest : public ::testing::Test
+{
+protected:
+    void SetUp() override
+    {
+        path image_directory = (path("../") / path("test") / path("fixtures") /
+                                path("panorama_aus_1"));
+
+        std::list<GeoImage> input {
+            GeoImage::fromExif((image_directory / path("P5050970.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5060971.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5060972.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5070973.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5070974.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5080975.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5080976.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5090977.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5090978.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5100979.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5100980.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5110981.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5110982.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5120983.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5120984.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5130985.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5130986.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5130987.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5140988.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5140989.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5150990.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5150991.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5160992.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5160993.JPG")).string()),
+            GeoImage::fromExif((image_directory / path("P5160994.JPG")).string())
+        };
```

Comment:
It can be difficult to debug this if one of the files is broken in some fashion. Consider rewriting this in a loop.

```suggestion
        std::list<GeoImage> input;
        const auto &dir = image_directory;
        for (const auto& p : {
            path{dir / "a.jpg"},
            path(dir / "b.jpg"},
            ... etc
        }) {
            auto img = GeoImage::fromExif(p.string());
            input.push_back(std::move(img));
        }
```

With that change it becomes a lot easier to breakpoint on a specific troublesome file.

Reply:
Yeah, I really disliked this and wanted to rename the files, but my hesitation is that it doesn't look like I can just rename the files because they're in LFS.  From what I've read, you can only delete and re-add the files with new names, but that doesn't free the storage taken by the old files.  I got as far as reading suggestions to delete the repo and recreate it before I decided that it wasn't worth it.

That said, I'm definitely in favor of doing this for multiple reasons if y'all are okay with that.  We'd lose this PR history, but better now than months from now.

Reply:
Per discussion in scrum sync;

You shouldn't need to rename files. They're not named "1", "2", etc but that shouldn't be a big deal since they do appear to be in ascending order anyway. I'm just looking to be able to breakpoint on `GeoImage::fromExif` and possibly loop a few times. That's easy with a loop and a lot harder if it's all done in a single statement.

Reply:
Done.